pkgs/top-level/all-packages.nix

```diff
@@ -26563,6 +26563,8 @@ with pkgs;
 
   felix_remoteshell = callPackage ../servers/felix/remoteshell.nix { };
 
+  ffsubsync = callPackage ../tools/video/ffsubsync { };
```

Comment:
Use the new [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format to avoid having to edit this file